### PR TITLE
Change violinplot to xarray

### PR DESCRIPTION
**Pull request by canyon289**
Fix for https://github.com/arviz-devs/arviz/issues/130

WIP Traveling and don't have access to my linux desktop so can't locally. I think things are ok but TravisCI will tell me for sure

### CODE REVIEW

**Comment by aloctavodia** [comment]
LGTM!

File: arviz/plots/violintraceplot.py [hunk 1]
```diff
@@ -76,11 +80,10 @@ def violintraceplot(trace, varnames=None, quartiles=True, credible_interval=0.94
         ax[axind].plot([0, 0], hpd_intervals, lw=linewidth, color='k', solid_capstyle='round')
         ax[axind].plot(0, per[-1], 'wo', ms=linewidth*1.5)
 
-        ax[axind].set_xlabel(names[axind], fontsize=textsize)
+        ax[axind].set_xlabel(var_name, fontsize=textsize)
```
Comment (ColCarroll) [inline arviz/plots/violintraceplot.py]: I'd use `for axind, (var_name, selection, x) in enumerate(plotters):` above, and then use `make_label(var_name, selection)` here.

**Comment by canyon289** [comment]
@ColCarroll Thanks for the review. I'll update the tests and rebase when I get back to my home next Wednesday. Only have my mac right now so  I can't test

**Comment by canyon289** [comment]
I think this is good to go now!

**Comment by canyon289** [comment]
Except for a rebase, one second

**Pull request merged**